unstructured_inference/models/tables.py

```diff
@@ -118,9 +119,13 @@ def get_tokens(self, x: Image):
                 text_height < inference_config.TESSERACT_MIN_TEXT_HEIGHT
                 or text_height > inference_config.TESSERACT_MAX_TEXT_HEIGHT
             ):
+                max_zoom = max(0, np.round(np.sqrt(TESSERACT_MAX_SIZE / np.prod(x.size) / 32), 1))
                 # rounding avoids unnecessary precision and potential numerical issues assocaited
                 # with numbers very close to 1 inside cv2 image processing
-                zoom = np.round(inference_config.TESSERACT_OPTIMUM_TEXT_HEIGHT / text_height, 1)
```

Comment:
We also has duplicated code for this in [unst](https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/pdf_image/ocr.py#L464) and `get_tokens` method is not used in unst after OCR refactor. Are we ready to deprecate the code in inference? cc @cragwolfe @qued

Reply:
yes, we definitely want to move OCR code for tables to unstructured from unstructured-inference

Reply:
@badGarnet lets move this to unst and i will deprecate all table OCR code in inference in separate PR :)

Reply:
that's true; we can finish refactor first; and I will make a PR on unstructured repo